ci-operator/step-registry/dpu-operator/e2e-tests-workflow/tests/dpu-operator-e2e-tests-workflow-tests-commands.sh

```diff
@@ -16,10 +16,10 @@ wait_for_job_to_run() {
 	do
 		JSON_FILE=$(mktemp /tmp/pullnumber.XXX)
 		curl -k --resolve "$endpoint_resolve" -X GET $1/api/json > $JSON_FILE
-		blocked=$(cat $JSON_FILE | jq .blocked)
-		if [[ "$blocked" == "true" ]]; then
+		blocked=$(cat $JSON_FILE | jq .why)
+		if [[ "$blocked" == "\"Waiting for next available executor on ‘ipu’\"" ]]; then
```

Comment:
Is it possible to do a generic regex instead of hardcoding "ipu"? I feel like if we change this, we will forget to change it here.